integrations/pagerduty.py

```diff
@@ -0,0 +1,249 @@
+    """
+        Log the message in the log file with the timestamp, if debug flag
+        is enabled
+
+        Parameters
+        ----------
+        msg : str
+            The message to be logged.
+    """
+    if debug_enabled:
+        msg = "{0}: {1}\n".format(now, msg)
+        print(msg)
+        with open(LOG_FILE, "a") as f:
+            f.write(msg)
+
+
+def generate_msg(alert: any, options: any, apikey: str) -> str:
+    """
+        Generate the JSON object with the message to be send
+
+        Parameters
+        ----------
+        alert : any
+            JSON alert object.
+        options: any
+            JSON options object.
+
+        Returns
+        -------
+        msg: str
+            The JSON message to send
+    """
+    managed_security_url    = 'https://wazuh.com'
+    level                   = alert['rule']['level']
+
+    severity = 'info'
+    if level >= 7:
+        severity = 'warning'
+    elif level >= 10:
+        severity = 'error'
+    elif level >= 13:
+        severity = 'critical'
+
+    groups = ', '.join(alert['rule']['groups'])
+
+    msg = {
+        'routing_key':  apikey,
+        'event_action': 'trigger',
+        'payload': {
+            "summary": alert['rule']['description'] if 'description' in alert['rule'] else "N/A",
+            "timestamp": alert['timestamp'],
+            "source": alert['agent']['location'],
+            "severity": severity,
+            "group": groups,
+            "custom_details": alert
+        },
+        "client": "Wazuh Monitoring Service",
+        "client_url": managed_security_url
+    }
+
+    if(options):
+        msg.update(options)
+
+    return json.dumps(msg)
+
+def send_msg(msg: any) -> None:
+    """
+        Send the message to the API
+
+        Parameters
+        ----------
+        msg : str
+            JSON message.
+        url: str
+            URL of the API.
+    """
+
+    headers = {'content-type': 'application/json', 'Accept-Charset': 'UTF-8'}
+    url     = 'https://events.pagerduty.com/v2/enqueue'
+    res     = requests.post(url, data=msg, headers=headers)
+    debug("# Response received: %s" % res)
+
+def get_json_file(file_location: str) -> any:
+    """
+        Read the JSON object from alert file
+
+        Parameters
+        ----------
+        file_location : str
+            Path to file alert location.
+
+        Returns
+        -------
+        {}: any
+            The JSON object read it.
+
+        Raises
+        ------
+        FileNotFoundError
+            If no alert file is not present.
+        JSONDecodeError
+            If no valid JSON file are used
+    """
+    try:
+        with open(file_location) as alert_file:
+            return json.load(alert_file)
+    except FileNotFoundError:
+        debug("# JSON file %s doesn't exist" % file_location)
+        sys.exit(ERR_FILE_NOT_FOUND)
+    except json.decoder.JSONDecodeError as e:
+        debug("Failed getting json_alert %s" % e)
```

Comment:
Modify debug message to be generic.
```suggestion
        debug("Failed getting JSON file. Error: %s" % e)
```
Change in all integrations.